pkg/services/user/userimpl/user.go

```diff
@@ -158,11 +153,10 @@ func (s *Service) Create(ctx context.Context, cmd *user.CreateUserCommand) (*use
 		}
 		_, err = s.orgService.InsertOrgUser(ctx, &orgUser)
 		if err != nil {
-			err := s.store.Delete(ctx, userID)
+			err := s.store.Delete(ctx, usr.ID)
```

Comment:
Are we sure that the `ID` is set here?

Reply:
I believe so, because `Insert` should have returned an error if it wasn't able to add the record. I could follow the insert call with a GetUserByLogin call to confirm, if you'd prefer.